R/data.table.R

```diff
@@ -2624,7 +2619,7 @@ setnames = function(x,old,new,skip_absent=FALSE) {
 setcolorder = function(x, neworder=key(x))
 {
   if (is.character(neworder) && anyDuplicated(names(x)))
-    stop("x has some duplicated column name(s): ", paste(names(x)[duplicated(names(x))], collapse=","), ". Please remove or rename the duplicate(s) and try again.")
+    stopf("x has some duplicated column name(s): %s. Please remove or rename the duplicate(s) and try again.", brackify(unique(names(x)[duplicated(names(x))])))
```

Comment:
the new `unique()` call can be dropped because `duplicated()` already returns a unique set, iiuc?

Reply:
NVM. Quite right to add the `unique()`.